api/common/configentry.go

```diff
@@ -57,4 +57,6 @@ type ConfigEntryResource interface {
 	DeepCopyObject() runtime.Object
 	// Validate returns an error if the resource is invalid.
 	Validate(namespacesEnabled bool) error
+	// Default sets the namespace field on the config entry spec to their default values if namespaces are enabled.
+	Default(consulNamespacesEnabled bool, destinationNamespace string, mirroring bool, prefix string)
```

Comment:
Should we call this `DefaultNamespaceFields`?

Reply:
Sounds good